code/controllers/subsystem/changelog.dm

```diff
@@ -15,9 +15,9 @@ SUBSYSTEM_DEF(changelog)
 	flags = SS_NO_FIRE
 	var/current_cl_timestamp = "0" // Timestamp is seconds since UNIX epoch (1st January 1970). ITs also a string because BYOND doesnt like big numbers.
 	var/ss_ready = FALSE // Is the SS ready? We dont want to run procs if we have not generated yet
-	var/list/startup_clients_button = list() // Clients who connected before initialization who need their button color updating
-	var/list/startup_clients_open = list() // Clients who connected before initialization who need the CL opening
-	var/changelogHTML = "" // HTML that the changelog will use to display
+	var/list/client/startup_clients_button = list() // Clients who connected before initialization who need their button color updating
```

Comment:
Ain't this just the entire client list? Since this is called when newing a client

Reply:
Look I wrote back in 2019 ok

Reply:
Hahaha. Live and improve eh? :P